observability/logging/logging_release_notes/cluster-logging-collector.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: cluster-logging-collector
+[id="cluster-logging-collector"]
+= Configuring the logging collector
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+{logging-title-uc} collects operations and application logs from your cluster and enriches the data with Kubernetes pod and project metadata.
+All supported modifications to the log collector can be performed though the `spec.collection` stanza in the `ClusterLogging` custom resource (CR).
+
+include::modules/configuring-logging-collector.adoc[leveloffset=+1]
+
+include::modules/creating-logfilesmetricexporter.adoc[leveloffset=+1]
+
+include::modules/cluster-logging-collector-limits.adoc[leveloffset=+1]
+
+[id="cluster-logging-collector-input-receivers_{context}"]
+== Configuring input receivers
+
+The {clo} deploys a service for each configured input receiver so that clients can write to the collector. This service exposes the port specified for the input receiver.
+The service name is generated based on the following:
```

Comment:
```suggestion
The service name is generated based on the following cases:
```

OR situations/conditions/scenarios depends what suits more here.

Reply:
chaned to "The service name is generated as follows:", WDYT?

Reply:
sounds good